insights/parsers/sockstat.py

```diff
@@ -0,0 +1,117 @@
+"""
+SockStats - file ``/proc/net/sockstat``
+=======================================
+The ``TcpIpStats`` class implements the parsing of ``/proc/net/snmp``
+file, which contains TCP/IP stats of individual layer.
+"""
+
+
+from .. import Parser, parser, LegacyItemAccess
+from insights.parsers import SkipException
+from insights.specs import Specs
+
+
+@parser(Specs.sockstat)
+class SockStats(Parser, LegacyItemAccess):
```

Comment:
The `LegacyItemAccess` only works with `self.data` which is not in your class. In fact, the `LegacyItemAccess` will be deprecated soon. If you want the Parser can be accessed like a `dict`, you can mix it with `dict` directly.

FYI: https://github.com/RedHatInsights/insights-core/blob/89828b760399231917414c8326515fc97d79118c/insights/parsers/modinfo.py#L145

Reply:
Done